cli.js

```diff
@@ -65,7 +69,25 @@ process.stdin
 		const geojson = JSON.parse(input)
 		const [west, south, east, north] = bbox(geojson)
 
-		const polylines = svgify(geojson, {projection: project})
+		const polylines = svgify(geojson, {
+			projection: project,
+			computeProps(feature) {
+				const {properties} = feature
+				const dataAttributes = Object.entries(properties).reduce((acc, entry) => {
+					const [key, value] = entry
+					if (!propertiesArray.includes(key)) return acc;
+
+					const dataKey = `data-${key.toLowerCase().replace('_', '-')}`
```

Comment:
Please keep the original key name.

Reply:
The name of the data attributes must comply with this spec:
Embedding custom non-visible data with the data-* attributes
A custom data attribute is an attribute in no namespace whose name starts with the string "data-", has at least one character after the hyphen, is XML-compatible, and contains no uppercase ASCII letters.
https://stackoverflow.com/questions/19533402/naming-rules-for-html5-custom-data-attributedata

Reply:
Then let's do `key.toLowerCase()` only. My intention is to keep `geojson-to-svg` as least magic as possible.